test/SlowTests/Issues/RavenDB-17096.cs

```diff
@@ -103,8 +103,16 @@ public async Task EtlFromReAddedNodeShouldWork()
                 Assert.True(etlDone.Wait(TimeSpan.FromSeconds(10)));
 
                 var addResult = await src.Maintenance.Server.SendAsync(new AddDatabaseNodeOperation(src.Database, node: mentorTag));
-                await WaitForRaftIndexToBeAppliedInCluster(addResult.RaftCommandIndex, TimeSpan.FromSeconds(30));
-                await WaitAndAssertForValueAsync(() => GetMembersCount(src), 3);
+                Assert.Equal(2, addResult.Topology.Members.Count);
+                Assert.Equal(1, addResult.Topology.Promotables.Count);
+
+                await WaitForRaftIndexToBeAppliedInCluster(addResult.RaftCommandIndex, TimeSpan.FromSeconds(15));
+                await Task.Delay(TimeSpan.FromSeconds(5)); // wait for the observer
```

Comment:
Can we avoid this? Is there some event that we need to wait for? Can we try to pull the value? Could you explain why we need this?

Reply:
removed